Documentation/cmdref/cilium-agent.md

```diff
@@ -37,6 +37,7 @@ cilium-agent [flags]
       --bpf-lb-dev-ip-addr-inherit string                    Device name which IP addr is inherited by devices running LB BPF program (--devices)
       --bpf-lb-dsr-dispatch string                           BPF load balancing DSR dispatch method ("opt", "ipip") (default "opt")
       --bpf-lb-dsr-l4-xlate string                           BPF load balancing DSR L4 DNAT method for IPIP ("frontend", "backend") (default "frontend")
+      --bpf-lb-external-clusterip                            Enable external access to ClusterIP services
```

Comment:
small doc nit, but no need to rerun ci: could we append here: `(default false)` ?